prime-router/settings/prod/0009-nj-doh-elr.yml

```diff
@@ -0,0 +1,16 @@
+# NJ DOH ELR connection
+name: elr
+organizationName: nj-doh
+topic: covid-19
+jurisdictionalFilter:
+  - matches(ordering_facility_state, NJ)
+translation:
```

Comment:
Your call, but I think, unless the state requests otherwise, by default we should match on patient_state as well.